README.md

```diff
@@ -1,7 +1,30 @@
 # Microsoft Linux Package Repositories
-This repository is intended to communicate status of [packages.microsoft.com](https://packages.microsoft.com) and to receive feedback on the service. Issues can be filed in the [issues](https://github.com/microsoft/linux-package-reposotories/issues) page. For security issues, refer to [SECURITY.md](SECURITY.md).
 
-## Contributing
+## Overview 
+
+The Microsoft Linux Package Repositories are hosted on PMC ([packages.microsoft.com](https://packages.microsoft.com)) service. The PMC service is intended to support package hosting for customers with clients running a distribution of the Linux OS. Microsoft builds and supports a variety of software products for Linux systems and makes them available via standard APT and YUM package repositories.  
```

Comment:
recommend "distribution of Linux."
Avoids argument of whether Linux is OS vs Kernel.